.env.example

```
@@ -0,0 +1,47 @@
+# Database
+DATABASE_URL="postgresql://eduboards:eduboards_dev@localhost:5432/eduboards"
```

dotenv-linter (3.3.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)